Sources/Utility/ArgumentParser.swift

```diff
@@ -383,6 +404,11 @@ public final class ArgumentParser {
         return arg
     }
     
+    /// Add a parser with a subcommand name.
+    public func add(subparser command: String, parser: ArgumentParser) {
```

Comment:
At the moment it is used to create the "root" argument parser, for generating any completions after `swift `: tools and compiler arguments.